staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -4729,7 +4729,7 @@ func printAnnotationsMultiline(w PrefixWriter, title string, annotations map[str
 		}
 		keys = append(keys, key)
 	}
-	if len(annotations) == 0 {
+	if len(keys) == 0 {
```

Comment:
Above logic in line 4719 could be deleted:
```
	if len(annotations) == 0 {
		w.WriteLine("<none>")
		return
	}
```

Reply:
@zhouya0 
Yep, intentionally kept the changes very minimal since it's my first contribution. I've removed the above redundant code, let me know if it looks good now. Thanks!